soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
Whoops, I think my grep tool missed them for some reason
[one](https://github.com/RoboJackets/robocup-software/blob/3ed28930ceb81f271784b00eb55f36c87e28d7d9/soccer/gameplay/tactics/defense.py#L112) [two](https://github.com/RoboJackets/robocup-software/blob/master/soccer/gameplay/tactics/positions/defender.py#L199).

@chachmu can you restore the previous setter, in case there is an edge case somewhere?